nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -159,6 +159,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <CsWinRTInternalProjection Condition="'$(CsWinRTPrivateProjection)' == 'true'">-internal</CsWinRTInternalProjection>
       <CsWinRTEmbeddedProjection Condition="'$(CsWinRTEmbedded)' == 'true'">-embedded</CsWinRTEmbeddedProjection>
       <CsWinRTEmbeddedEnums Condition="'$(CsWinRTEmbeddedPublicEnums)' == 'true'">-public_enums</CsWinRTEmbeddedEnums>
+
```

Comment:
You can probably simplify this and do this in one place by moving setting `CsWinRTExeTFM` out of the `CsWinRTGenerateProjection` target and instead to the top in the common property group.